package.json

```diff
@@ -122,38 +122,37 @@
     "react-async-component": "0.2.2",
     "react-dom": "15.4.2",
     "react-helmet": "4.0.0",
-    "react-router-dom": "4.0.0-beta.7",
+    "react-router-dom": "4.0.0",
     "serialize-javascript": "1.3.0",
     "uuid": "3.0.1"
   },
   "devDependencies": {
     "assets-webpack-plugin": "3.5.1",
-    "babel-cli": "6.23.0",
-    "babel-core": "6.23.1",
+    "babel-cli": "6.24.0",
+    "babel-core": "6.24.0",
     "babel-eslint": "7.1.1",
     "babel-jest": "19.0.0",
-    "babel-loader": "6.3.2",
+    "babel-loader": "6.4.0",
     "babel-plugin-transform-react-constant-elements": "6.23.0",
     "babel-plugin-transform-react-inline-elements": "6.22.0",
     "babel-plugin-transform-react-jsx-self": "6.22.0",
     "babel-plugin-transform-react-jsx-source": "6.22.0",
     "babel-polyfill": "6.23.0",
-    "babel-preset-env": "1.2.0",
-    "babel-preset-latest": "6.22.0",
```

Comment:
done in [1159a18](https://github.com/ctrlplusb/react-universally/pull/396/commits/1159a1853162ad9a828358820cad5dd1b171ab92)